pkgs/development/libraries/boost/generic.nix

```diff
@@ -289,6 +289,20 @@ stdenv.mkDerivation {
       EOF
     '';
 
+  # Fix compilation to 32-bit ARM with clang in downstream packages
+  # https://github.com/ned14/outcome/pull/308
+  # https://github.com/boostorg/json/pull/1064
+  postPatch = lib.optionalString (lib.versionAtLeast version "1.87") ''
```

Comment:
Since the PR is merged, why not (majorMinor version) == "1.87"? (With a comment at the boost187 definition site to remove the patch once we drop that version)

Reply:
why not just drop the patch once the version is > 1.87?

Reply:
Yes, that would be `(lib.versionOlder version "1.88")`. If the patch applies to all previous versions.

Reply:
Sounds good to me, then. We can revisit once things land.